lib/core/src/lib/notifications/components/notification-history.component.ts

```diff
@@ -112,9 +118,14 @@ export class NotificationHistoryComponent implements OnDestroy, OnInit, AfterVie
     }
 
     markAsRead() {
-        this.notifications = [];
+        this.unreadNotifications = [];
+        this.notifications.forEach((notification: NotificationModel) => {
+            notification['read'] = true;
```

Comment:
Please refer to read property by .read not by ['read']

Reply:
updated to use .read